eslint_test.yaml

```diff
@@ -1,9 +1,6 @@
-parser: babel-eslint
+parser: "@babel/eslint-parser"
```

Comment:
Suggestion: Please document the parser decision here.

Reply:
I sort of fixed this, and then almost immediately realised I could & should fix it better. The `@babel/eslint-parser` dependency is now only in `fluent-react/`, as it's really only required for the JSX content in its test files.

For the `import.meta` use in other test files, it's sufficient to use `ecmaVersion: 2020` as a parser option.